pkg/lint/load.go

```diff
@@ -75,6 +75,11 @@ func (cl *ContextLoader) findLoadMode(linters []*linter.Config) packages.LoadMod
 
 func (cl *ContextLoader) buildArgs() []string {
 	args := cl.cfg.Run.Args
+	// disable vcs stamping to avoid git exit status 128 errors
+	if cl.cfg.Run.Go == "1.18" && len(args) == 0 {
```

Comment:
what about 1.19, 1.20, etc?

Reply:
good question, maybe it makes sense to take into account unreleased versions, but how far?

Reply:
No, I just highlighted the fragility of the changes. Is it possible to check "feature" in runtime?

> This new go feature breaks `go list ./...`

Could someone provide details? Maybe a bug in [golang/go](https://github.com/golang/go) or a reference to the docs?